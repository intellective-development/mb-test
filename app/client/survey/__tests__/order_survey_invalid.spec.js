import * as React from 'react';
import OrderSurveyInvalid from '../order_survey_invalid';

it('renders', () => {
  expect(render(
    <OrderSurveyInvalid />
  )).toMatchSnapshot();
});
