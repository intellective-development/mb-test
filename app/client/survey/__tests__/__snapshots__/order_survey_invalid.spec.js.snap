// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  class="checkout-frame no-bottom-margin"
>
  <div
    class="modal-header mega-header"
  >
    <h3>
      Sorry, the link you followed was invalid
    </h3>
    <h2>
      Go to 
      <a
        href="/"
      >
        minibardelivery.com
      </a>
    </h2>
  </div>
</div>
`;
